packages/core/src/future.ts

```diff
@@ -23,8 +23,9 @@ export interface Future<T> extends Promise<T>, FutureLike<T> {
 export interface NewFuture<T> {
   future: Future<T>;
   produce(value: Value<T>): void;
-  /** @deprecated Use produce(value) instead */
```

Comment:
It might be an idea to keep the `/** @deprecated Use produce(value) instead */`.  It tells the language server to put a strike through into language server aware IDEs

![Screenshot 2021-08-24 at 14 30 34](https://user-images.githubusercontent.com/118328/130625538-8813ac96-0eef-4986-9ee1-d5675c850e94.png)